posts/2023-09-01-electing-new-project-directors.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Electing New Project Directors"
+author: Leadership Council
+team: Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>
+---
+
+Today we are launching the process to elect new Project Directors to the Rust Foundation Board of Directors.
+As we begin the process, we wanted to spend some time explaining the goals and procedures we will follow.
+We will summarize everything here, but if you would like to you can read the [official process documentation][pde-process].
+
+We ask all project members to begin working with their Leadership Council representative to nominate potential Project Directors. See the [Candidate Gathering](#candidate-gathering) section for more details. Nominations are due by September 15, 2023.
+
+<!-- TODO: Once we've merged this PR we should link to the final location instead. -->
+[pde-process]: https://github.com/rust-lang/leadership-council/pull/12
+
+## What are Project Directors?
+
+The Rust Foundation Board of Directors has five seats reserved for Project Directors.
+These Project Directors serve as representatives of the Rust project itself on the Board.
+Like all Directors, the Project Directors are elected by the entity they represent, which in the case of the Rust Project means they are elected by the Rust Leadership Council.
+Project Directors serve for a term of two years and will have staggered terms.
+This year we will appoint two new directors and next year we will appoint three new directors.
+
+We want to make sure the Project Directors can effectively represent the project as a whole, so we are soliciting input from the whole project.
+The elections process will go through two phases: Candidate Gathering and Election.
+Read on for more detail about how these work.
+
+## Candidate Gathering
+
+The first phase is beginning right now.
+In this phase, we are inviting the members of the Rust Project to nominate people who will make good project directors.

```

Comment:
Given that one of our goals is to clarify what "members of the Rust Project" means, I think we should say explicitly here that we're meaning "members of a Rust team"

Reply:
I'll reword this as "members of all of the top level Rust teams and their subteams"